package-lock.json

```diff
@@ -5,6 +5,7 @@
   "requires": true,
   "packages": {
     "": {
+      "name": "mattermost-webapp",
```

Comment:
I think we want to undo this package-lock.json change. I thought I remembered some discussion in mattermost community about this getting auto-removed and added, but can't seem to find it. Not sure what the consensus was or if we even found why some developers have it added and others have it removed. I'd guess something about the npm version they are using locally?

Reply:
We should actually always have that line, so that change was correct. Someone must be on an older NPM version that keeps removing it...